mmv1/products/identityplatform/api.yaml

```diff
@@ -505,3 +505,94 @@ objects:
           Whether authentication is disabled for the tenant. If true, the users under
           the disabled tenant are not allowed to sign-in. Admins of the disabled tenant
           are not able to manage its users.
+  - !ruby/object:Api::Resource
+    name: ProjectDefaultConfig
```

Comment:
I'm not super familiar with this resource, but it seems this may be the same resource as line 31 above. Is there a reason we want to separate them?

Reply:
Actually, Stephen suggested that the scope of this issue is to create a new resource. You can see the conversation in this issue; https://github.com/hashicorp/terraform-provider-google/issues/8288#issuecomment-1261287976.